CMakeLists.txt

```diff
@@ -712,7 +712,11 @@ endif()
 # tarantool info summary (used in server version output)
 #
 set(TARANTOOL_OPTIONS "-DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}")
-set(TARANTOOL_OPTIONS "${TARANTOOL_OPTIONS} -DENABLE_BACKTRACE=${ENABLE_BACKTRACE}")
+if(ENABLE_BACKTRACE)
+    set(TARANTOOL_OPTIONS "${TARANTOOL_OPTIONS} -DENABLE_BACKTRACE=TRUE")
+else()
+    set(TARANTOOL_OPTIONS "${TARANTOOL_OPTIONS} -DENABLE_BACKTRACE=FALSE")
+endif()
```

Comment:
I looked over tarantool and its submodules (curl, luajit, libyaml, zstd, nghttp2) and it seems that ON/OFF is usually used in `option()` declarations. So I guess it is a kind of unofficial canonical form for boolean options.

Feel free to ignore it is just my guess.